scripts/workers/submission_worker.py

```diff
@@ -519,7 +529,6 @@ def main():
 
     while True:
         for message in queue.receive_messages():
-            # Print out the body of the message
```

Comment:
Why we are removing this comment in this PR?

Reply:
This comment is pretty much useless. You can directly figure out what is going on in that line and don't need this comment.

Reply:
I know, just wanted to highlight it so that we won't wonder about it in future.
Rest Looks goo!